### PR TITLE
Get authentication headers from default secret from CCE/ServiceStage

### PR DESCRIPTION
**Pull request by humingcheng**

### CODE REVIEW

File: adaptor/gochassis/plugin.go [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+//DefaultRefreshInterval is default refresh interval
+const DefaultRefreshInterval = 60 * time.Second
+
+//variables below is to make the package more testable
+
+//ServiceAccountPath is where the token/namespace file exists in k8s container
+var ServiceAccountPath = `/var/run/secrets/kubernetes.io/serviceaccount`
+
+//ServiceStageMountPath is where the ServiceStage mount the default file
+var ServiceStageMountPath = `/opt/CSE/etc/auth`
```
Comment (tianxiaoliang) [inline adaptor/gochassis/plugin.go]: var to const

File: adaptor/gochassis/plugin.go [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+	DefaultSecretFile = `.dockerconfigjson`
+)
+
+//DefaultRefreshInterval is default refresh interval
+const DefaultRefreshInterval = 60 * time.Second
+
+//variables below is to make the package more testable
+
+//ServiceAccountPath is where the token/namespace file exists in k8s container
+var ServiceAccountPath = `/var/run/secrets/kubernetes.io/serviceaccount`
```
Comment (tianxiaoliang) [inline adaptor/gochassis/plugin.go]: const

File: adaptor/gochassis/plugin.go (hunk 1 above)
Comment (humingcheng) [reply adaptor/gochassis/plugin.go]: Done

File: adaptor/gochassis/plugin.go (hunk 2 above)
Comment (humingcheng) [reply adaptor/gochassis/plugin.go]: Done

**Pull request merged**